File: src/main/java/omerozturk/hrms/business/abstracts/EmployeeService.java
package omerozturk.hrms.business.abstracts;

import omerozturk.hrms.core.utilities.result.DataResult;
import omerozturk.hrms.core.utilities.result.Result;
import omerozturk.hrms.entities.concretes.Employee;

import java.util.List;

public interface EmployeeService {
    DataResult<List<Employee>> getAll();
    Result add(Employee employee);
    Result delete(Employee employee);
    DataResult<Employee> getById(int employeeId);
}

File: src/main/java/omerozturk/hrms/uiPresentation/src/services/employeeServices/employeeService.js
import axios from "axios";

export default class EmployeeService {
    getAll() {
        return axios.get("http://localhost:8080/api/employees/getall" );
    }
    getByEmployeeId() {
        return axios.get("http://localhost:8080/api/employees/getbyid?employeeId=1");
    }

}